src/test/miner_tests.cpp

```diff
@@ -409,8 +409,8 @@ BOOST_AUTO_TEST_CASE(CreateNewBlock_validity)
     }
 
     // non-final txs in mempool
-    SetMockTime(m_node.chainman->ActiveChain().Tip()->GetMedianTimePast()+1);
-    const int flags{LOCKTIME_VERIFY_SEQUENCE | LOCKTIME_MEDIAN_TIME_PAST};
+    SetMockTime(m_node.chainman->ActiveChain().Tip()->GetMedianTimePast() + 1);
+    const int flags{LOCKTIME_VERIFY_SEQUENCE};
```

Comment:
Shouldn't this be `STANDARD_LOCKTIME_VERIFY_FLAGS` instead?

Reply:
No idea. `STANDARD_LOCKTIME_VERIFY_FLAGS` wasn't picked for this test when this was added in commit c6c2f0fd782ccf607027414012f45c8f48561a30 , so I'll leave this as-is for now. Unless there is a reason to change.